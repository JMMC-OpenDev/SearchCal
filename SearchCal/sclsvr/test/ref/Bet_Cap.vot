version https://git-lfs.github.com/spec/v1
oid sha256:bd04b90d343f40536a162f5b24ca7436bbadf2e7651c9228b8bf559496d5eca1
size 146546
