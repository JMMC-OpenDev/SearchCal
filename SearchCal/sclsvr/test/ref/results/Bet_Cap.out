version https://git-lfs.github.com/spec/v1
oid sha256:58f1524e5e26f8e44aaf232dbf54e120e1288482523a0bed67ab6825ce238721
size 94113
